### ISSUE TITLE
Feature request: ATX Readout/control

### ISSUE THREAD

**Issue by AccaliaDeElementia** [adamoutler/pikvm-homeassistant-integration#7]
Feature request:

Ability to read out ATX power state (hard drive state optionally) and expose toggle/switch in HomeAssistant to power on/off and perform short/long presses of the buttons.

This would be very handy to have and with the API already implemented on the PiKVM side it hopefully won't be a difficult lift to implement. right? :)

**Comment by adamoutler** [comment]
Yes.  This is for sure a planned feature.   However, right now I'm waiting for ZeroConf features in next PiKVM release.  The ZeroConf feature will provide a good user experience for adding and maintaining dynamic IP PiKVM devices.  

Once completed, I will begin shaking out any problems and open a new repository for merging with Home Assistant core.  They recommend starting with Sensor domains only and I'm going to heed that warning.  

While the GPIOs are 1/2 sensors, it would be feature complete without buttons to trigger the GPIOs.  And when that happens, people are also going to want Power/reset controls and ATX indicators.  Also, I'm going to add screenshot/video monitoring APIs as well. 

The idea will be to use this as a beta feature testing repository.  HA core integration will be the refined release version of this beta repo.  Lots coming, and implementation will be easy, but I'm in a long maintenance period right now.
